src/charm.py

```diff
@@ -65,7 +65,9 @@ def _mongodb_relation_joined(self, event) -> None:
                 event.defer()
                 return
 
-        # initialise replica set if not already done
+        # in future patch we will reconfigure the replicaset instead of re-initialising
+        # see: https://warthogs.atlassian.net/browse/DPE-64?atlOrigin=eyJpIjoiM2NmYWU2YWI5ZDkwNDQ1Nzg2ZWFmMjBkYjM5Y2VkMjMiLCJwIjoiaiJ9
```

Comment:
Is the `altOrigin` query string parameter needed here? Do you want to track something like described in https://developer.atlassian.com/developer-guide/client-identification/#origin-id-for-links-to-atlassian-ui? I also saw that this long line broke lint on CI.

Reply:
Yes! Opting for @paulomach suggestion here regarding the JIRA link